httomo/darks_flats.py

```diff
@@ -47,11 +47,18 @@ class DarksFlatsFileConfig(NamedTuple):
     3. Dark-field and flat-field images are stored in separate files with own unique or identical image keys.
     This can be a new dataset or two different dataset. Therefore, the image_key_path parameter should be provided
     for both flats and darks.
+
+    4. Dark-field or flat-field images are not available in the dataset, yet some processing of data is still required
+    (e.g., the case of applying distortion correction only).
+
+    5. Dark-field or flat-field images are available in the dataset but they need to be ignored, for instance to proceed
+    with the other types of data correction or reconstruction avoiding normalisation to d/f.
     """
 
     file: Path
     data_path: str
     image_key_path: Optional[str]
+    ignore: bool
```

Comment:
Small suggestion if you're interested, feel free to take it or leave it.

It seems like many times the `ignore` parameter is used, it's given a value of `False`, and that has caused a decent number of tests to be changed, simply to pass `ignore=False`. This parameter could be given a default value of `False` to reduce the number of changes needed to fix all tests which create a `DarksFlatsFileConfig` object but are simply passing `ignore=False` anyway.